thorlcr/slave/slavmain.cpp

```diff
@@ -1214,21 +1173,60 @@ class CKJService : public CSimpleInterfaceOf<IKJService>, implements IThreaded,
     void addToKeyManagerCache(CKMContainer *kmc)
     {
         CriticalBlock b(kMCrit);
```

Comment:
This, and the other cache code, should cope with maxCachedKJManagers == 0, since it can be set from an option, and that would crash the following code.
It might also be useful so you can compare the effect of having no cache...